### DISCUSSION TITLE
dynamic config load and support multi config source ,eg mysql

### DISCUSSION THREAD

**Discussion by huhan110** [prometheus/prometheus#10170] [Ideas]
## Proposal
**Use case. Why is this important?**

*“Nice to have” is not a good use case. :)*
Hope to support dynamic configuration loading and multi-configuration data source support, for example, configuration changes can be realized by adding and deleting configurations through mysql

**Comment by roidelapluie** [comment]
This is so far out of scope for Prometheus. Prometheus, as a monitoring solution, is started with files. However, you can define targets in many ways with service discovery, including generic SD like HTTP_SD.

I will move this to a github discussion, if other people want to comment.

**Comment by HaoKunT** [comment]
Are there any new developments in this proposal?
